README.md

```
@@ -34,6 +34,21 @@ Capsulo is a modern CMS that lets developers define content schemas in code whil
 npm install
 ```
 
+### Environment Configuration
+
+The CMS requires certain environment variables to function correctly, especially for GitHub authentication and optional Cloudflare R2 storage.
+
+1.  **Create a `.env` file** in the project root:
+    ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)